### PR TITLE
Remove unnecessary sleep for cluster starts

### PR DESCRIPTION
**Pull request by cdonati**

### CODE REVIEW

File: appscale/tools/remote_helper.py [hunk 1]
```diff
@@ -254,7 +254,7 @@ def start_head_node(cls, options, my_id, node_layout):
       if agent.PARAM_REGION in params:
         additional_params[agent.PARAM_REGION] = params[agent.PARAM_REGION]
 
-    time.sleep(10)  # gives machines in cloud extra time to boot up
+      time.sleep(10)  # gives machines in cloud extra time to boot up
```
Comment (obino) [inline appscale/tools/remote_helper.py]: Is this sleep actually necessary? And if so, do we know why 10 seconds is enough?

Comment (cdonati) [inline appscale/tools/remote_helper.py]: I doubt the sleep is necessary, but removing it for all cloud deployments would require a lot of testing.

Comment (menivaitsi) [inline appscale/tools/remote_helper.py]: I believe this was originally placed for GCE but was moved around and ended up here.

**Comment by cdonati** [comment]
https://ocd.appscale.com:8080/job/Daily%20Build/2808/

**Pull request merged**